functions.js

```diff
@@ -11,12 +11,11 @@ window.addEventListener("scroll", () => {
     : nav.classList.remove("c-nav--alt");
 });
 
-const pageVideo = document.getElementById("header__video");
-
-function playPause() {
+const pageVideo = document.querySelector(".header__video");
+document.querySelector(".button-border .play").addEventListener("click", () => {
```

Comment:
Why is this `".button-border .play"` rather than just `".play"`?

Reply:
Button border encapsulates play